exercises/practice/rotational-cipher/tests/rotational_cipher.cairo

```diff
@@ -0,0 +1,23 @@
+pub fn rotate(text: ByteArray, shift_key: u8) -> ByteArray {
```

Comment:
You seem to have accidentally copy/pasted the implementation instead of writing tests here :D

Reply:
This is why I shouldn’t code right before bed. :o